apps/next/src/components/StandaloneField/StandaloneField.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  FilledTextFieldProps,
+  TextField,
+  alpha,
+  styled,
+} from '@avalabs/k2-alpine';
+
+export type StandaloneFieldProps = Omit<
+  FilledTextFieldProps,
+  'variant' | 'size'
+>;
+
+export const StandaloneField = styled((props: StandaloneFieldProps) => (
+  <TextField
+    {...props}
+    variant="filled"
+    slotProps={{
+      ...props.slotProps,
+      input: {
+        ...props.slotProps?.input,
+        disableUnderline: true,
+      },
+    }}
+  />
+))(({ theme }) => ({
+  '& .MuiFilledInput-root': {
+    paddingBlock: 9,
+    paddingInline: 16,
+    overflow: 'hidden',
+    borderRadius: '25px',
+    border: 'none',
+    backgroundColor:
+      theme.palette.mode === 'light'
+        ? theme.palette.neutral['850_10']
+        : theme.palette.common.white_10,
```

Comment:
I found that instead of switching based on `palette.mode`, we can use `getHexAlpha(theme.palette.primary.main, XX)`